frontends/web/src/components/bitboxbase/connectbase.tsx

```diff
@@ -36,7 +36,7 @@ export class ConnectedBase extends Component<Props, State> {
     constructor(props) {
         super(props);
         this.state = {
-            blockInfo : '',
+            blockInfo : {},
```

Comment:
keep in mind and empty string `''` is falsy, but and empty object is truthy, this matters in the render when you check `if (!blockInfo) {`

![Screen Shot 2019-07-02 at 1 04 46 PM](https://user-images.githubusercontent.com/546900/60508146-1750aa00-9cca-11e9-82bb-c137a678230e.png)